client/.eslintrc

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
From the eslint website:

> JavaScript - use .eslintrc.js and export an object containing your configuration.
YAML - use .eslintrc.yaml or .eslintrc.yml to define the configuration structure.
JSON - use .eslintrc.json to define the configuration structure. ESLint’s JSON files also allow JavaScript-style comments.
__Deprecated - use .eslintrc, which can be either JSON or YAML.__
package.json - create an eslintConfig property in your package.json file and define your configuration there.

You may want to change your config file to a .js file like the CRA app has, seems like .eslint files are deprecated.

Reply:
Good catch